integration/emr/alluxio-emr.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
```

Comment:
can you document this script, especially adding comments on the arguments

Reply:
Done.